src/development/sic-calls/history.md

```diff
@@ -25,6 +26,38 @@
 - [Call #22: July 29, 2024](#call-22-july-29-2024)
 - [Call #21: July 15, 2024](#call-21-july-15-2024)
 
+## Call #43: October 06, 2025
+
+[Agenda](https://github.com/ethereum/pm/issues/1747)
+[Video recording](TBD)
+
+### Team updates
+- [@gballet](https://x.com/gballet) ([@go_ethereum](https://x.com/go_ethereum)) is assisting [@CPerezz19](https://x.com/CPerezz19) in reactivating execution spec tests to be binary-tree compatible and proposed a new gas-accounting PR (EIP-8032) aimed at stemming state growth.
+- [@CPerezz19](https://x.com/CPerezz19) is working with [@gballet](https://x.com/gballet) on binary-tree test fixes (state diffs and transitions), has identified and fixed bugs in the binary-tree implementation, and continues BloatNet work while pushing benchmarks to EELS for integration with Camille’s tool from [@NethermindEth](https://x.com/NethermindEth).
```

Comment:
actually the state diff part is what we did not work on, and after the rest of the talk, we will not do it.

Reply:
adjusted